### PR TITLE
feat: add globalfee initModule method for upgrade handler

### PR DESCRIPTION
**Pull request by yaruwangway**
# Add globalfee.InitModule

Add globalfee initModule for v8 upgrade Handler to use to init globalfee Params.

### CODE REVIEW

File: x/globalfee/module.go [hunk 1]
```diff
@@ -86,6 +86,12 @@ func NewAppModule(paramSpace paramstypes.Subspace) *AppModule {
 	return &AppModule{paramSpace: paramSpace}
 }
 
+// InitModule will initialize the globalfee module.
+//It should only be called once and as an alternative to InitGenesis.
```
Comment (glnro) [inline x/globalfee/module.go]:
```suggestion
// It should only be called once and as an alternative to InitGenesis.
```

**Review by glnro — APPROVED** [review]

File: x/globalfee/module.go [hunk 2]
```diff
@@ -86,6 +86,12 @@ func NewAppModule(paramSpace paramstypes.Subspace) *AppModule {
 	return &AppModule{paramSpace: paramSpace}
 }
 
+// InitModule will initialize the globalfee module.
+// It should only be called once and as an alternative to InitGenesis.
+func (a AppModule) InitModule(ctx sdk.Context, globalfeeParams types.Params) {
+	a.paramSpace.SetParamSet(ctx, &globalfeeParams)
```
Comment (glnro) [inline x/globalfee/module.go:92]: Do we want to do a nil check for `globalFeeParams` ?

Comment (yaruwangway) [reply x/globalfee/module.go:92]: good idea !

Comment (yaruwangway) [reply x/globalfee/module.go:92]: i think this cannot be nil... `globalfeeParams` is a struct.

**Pull request merged**